rllib/tests/test_lstm.py

```diff
@@ -84,6 +84,8 @@ class RNNSpyModel(Model):
     capture_index = 0
 
     def _build_layers_v2(self, input_dict, num_outputs, options):
+        # This is needed as cloudpickle will reference the same class.
+        RNNSpyModel.capture_index = 0
```

Comment:
cc @ericl

Reply:
I don't really understand the comment. Can you clarify if this fix addresses a cloudpickle bug or an RLlib bug and maybe say a bit more about it?

I saw the jenkins test failure and spent a little time looking into it, but didn't track it down successfully.

Reply:
It's neither a cloudpickle bug or a rllib bug; the behavior for serializing this test object has changed (but is a testing-specific behavior).

Reply:
updated comment